src/test/java/org/opensearch/security/dlic/rest/api/AccountApiTest.java

```diff
@@ -1,16 +1,16 @@
 /*
- * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
There is no need to change the existing license header. Just use the latest license header in the newly added files.